frontend/AnimeCore/src/lib/components/pages/anime_info/AnimeMainDetails.svelte

```diff
@@ -0,0 +1,210 @@
+<script lang="ts">
+	export let anime: any;
+
+	import ScrollArea from '$components/shared/ScrollArea.svelte';
+	import SidebarDetails from '$components/pages/anime_info/SidebarDetails.svelte';
+
+	// icons
+	import PlayCircle from '$icons/PlayCircle.svelte';
+	import Read from '$icons/Read.svelte';
+	import Listen from '$icons/Listen.svelte';
+	import Video from '$icons/Video.svelte';
+	import Download from '$icons/Download.svelte';
+	import Share from '$icons/Share.svelte';
+	import SettingsOutline from '$icons/SettingsOutline.svelte';
+	import Edit from '$icons/Edit.svelte';
+
+	const icon_mapping = {
+		anime_options: {
+			read: {
+				icon: {
+					component: Read,
+					width: 22,
+					height: 22,
+					color: "bg-surface-500"
+				}
+			},
+			listen: {
+				icon: {
+					component: Listen,
+					width: 22,
+					height: 22,
+					color: "bg-surface-500"
+				}
+			},
+		},
+		user_options_icons : {
+			video: {
+				icon: {
+					component: Video,
+					variant: false,
+					width: 20,
+					height: 20
+				}
+			},
+			edit: {
+				icon: {
+					component: Edit,
+					variant: "with_underline_around_pencil",
+					width: 20,
+					height: 20
+				}
+			},
+			download: {
+				icon: {
+					component: Download,
+					variant: false,
+					width: 20,
+					height: 20
+				}
+			},
+			share: {
+				icon: {
+					component: Share,
+					variant: false,
+					width: 20,
+					height: 20
+				}
+			}
+		}
+	}
+</script>
+
+<div class="grid grid-cols-12 items-start">
+	<div class="col-span-10 grid grid-cols-12 items-end md:gap-10">
+		<div class="col-span-7 flex items-end gap-10">
+			<img class="md:w-[15vw] md:rounded-xl" src={anime.cardBackgroundImage} alt={anime.titles.eng} />
+			<div>
+				<span class="font-bold md:text-[3vw]">{anime.titles.eng}</span>
+
+				<p class="flex items-center gap-2 text-surface-100">
+					<span class="md:text-[0.8vw] font-medium after:content-['▪'] after:ml-1">{anime.titles.japanese}</span>
```

Comment:
> after:content-['▪'] 

Use not last children css selector

Reply:
> Use not last children css selector

Why do we need that?

Reply:
> Why do we need that?

Less codes

Reply:
> Use not last children css selector

Honestly I didnt understand what you mean by using not last
Are saying to give after class to parent? If so its not working :|

Reply:
I meant [this](https://developer.mozilla.org/en-US/docs/Web/CSS/:last-of-type)

Reply:
They are css, is that oky

Reply:
Use tailwind queries

Reply:
https://github.com/baseplate-admin/CoreProject/pull/1871#discussion_r1161098128